lib/api/controllers/securityController.js

```diff
@@ -1,23 +1,48 @@
+/*
```

Comment:
I don't know if the licence preamble is mandatory in the beginning of all the source files, or if the `LICENCE.md` file at the project's root is enough.
But if this preamble is mandatory here, it should be in ALL the files (should be done in a dedicated PR, thus)